src/stores/cluster/index.js

```diff
@@ -122,6 +128,14 @@ export default class ClusterStore extends Store {
     // this.isLoading = false;
   };
 
+  @action
+  fetchJobs = async clusterId => {
```

Comment:
this `fetchJobs` already defined in cluster detail store

Reply:
ok